### PR TITLE
added a trim-strings option to trim string entry types during data import.

### PR DESCRIPTION
**Pull request by rishabhjain08**
solves issue #6331
set --trim-strings=true in import tool argument list to trim the fields which have string entry type.

### CODE REVIEW

**Comment by systay** [comment]
Hi @rishabhjain08!

Have you signed the CLA? http://neo4j.com/developer/cla/

That is why your build is failing. Also - the two commits seem to just be splitting production code and tests - maybe it's better to squash them into a single commit?

Regards,

Andrés

**Comment by rishabhjain08** [comment]
Sorry for that. I have squashed the commits and sent a signed CLA mail.

File: community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java [hunk 1]
```diff
@@ -134,6 +134,10 @@
                 "<true/false>",
                 "Whether or not fields from input source can span multiple lines, i.e. contain newline characters." ),
 
+        TRIM_STRINGS( "trim-strings", org.neo4j.csv.reader.Configuration.DEFAULT.trimStrings(),
+                "<true/false>",
+                "Whether or not fields with id-type string should be trimmed for whitespaces."),
```
Comment (tinwelint) [inline community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java]: This description is slightly misleading. It mentions that it trims fields with id-type string, but it seems that all fields of type string/string array are trimmed. `id-type` is a term for the `:ID`, `:START_ID`, `:END_ID` fields. It's probably enough saying something like:

`"Whether or not strings should be trimmed for whitespaces."`

**Comment by tinwelint** [comment]
Had a little comment about the documentation string. The changes and tests looks very good.

**Comment by rishabhjain08** [comment]
Have updated the message. Will I get a mail when my CLA is approved.

**Comment by tinwelint** [comment]
No, but I can see your sent mail, so all good. I'll poke the build to do another attempt.

**Comment by tinwelint** [comment]
@rishabhjain08 There seems to be a compilation error in `InputEntityDeserializer.java`

**Pull request merged**